lib/report_portal/cucumber/parallel_report.rb

```diff
@@ -105,8 +104,9 @@ def set_parallel_tests_vars
         end
       end
 
+      #time required for first tread to created remote project in RP and save id to file
       def wait_time_for_launch_start
-        60
+        ENV['rp_parallel_launch_wait_time'] ? ENV['rp_parallel_launch_wait_time'] : 60
```

Comment:
If you think this value should be configurable, it should be got from settings and documented in README

Reply:
I don't know what it's for was added by someone else, maybe to delay start on purpose

Reply:
currently getting this error on the remote box:
```
"ReportPortal API returned {\"error_code\":40025,\"message\":\"Start time of child ['Thu Jun 20 17:51:33 UTC 2019'] item should be same or later than start time ['Thu Jun 20 17:51:33 UTC 2019'] of the parent item/launch '5d0bc76a7405fd0001dbe5e2'\"}"
```